.github/workflows/build.yml

```diff
@@ -473,8 +481,9 @@ jobs:
               --enable-krbV-uam \
               --enable-pgp-uam \
               --without-afpstats \
+              --with-tracker-pkgconfig-version=1.0
```

Comment:
FWIW a fully updated Solaris 10.4 has v2.0 of the tracker tools.

Reply:
The existing VM is so old it only has Tracker 1.0